src/app/meetups/add-meetups/meetups-add.component.ts

```diff
@@ -16,7 +16,8 @@ import { switchMap } from 'rxjs/operators';
 import { findDocuments } from '../../shared/mangoQueries';
 
 @Component({
-  templateUrl: './meetups-add.component.html'
+  templateUrl: './meetups-add.component.html',
+  styles: [ 'mat-radio-button {margin:10px;}', 'mat-form-field.half-width {width:205pt;}' ]

```

Comment:
A few things here:
* Please reformat these styles to match other components, like:
```
styles: [ `
  mat-radio-button {
    margin:10px;
  }
` ]
```
* That `half-width` class is misleading since it hard codes what half width is rather than using a value relative to its parent.
* In the future, please do not use the `pt` unit and instead use `px` or `rem`